src/v/redpanda/admin_server.h

```diff
@@ -31,13 +31,19 @@
 
 #include <absl/container/flat_hash_map.h>
 
+#include <type_traits>
```

Comment:
nit: I guess this was for `std::is_enum_v`, which is only used in the implementation file.